### PR TITLE
Issue6 New terms in the glossary

### PR DESCRIPTION
**Pull request by milicag**
As discussed with Phil on Aug 1, we've been missing the following terms in the glossary:

1. Building Model
2. Control System
3. HVAC System
4. HVAC System Model
5. Non-HVAC System
6. Plant
7. Standard Control Sequence

I've also edited the terms in the ```useCases.rst``` in order to obtain consistency throughout the documentation. The definitions are relying on the newly developed use case from Phil and on the ```requirements.rst```.

For the edits in  ```useCases.rst``` please look for darker green edits, since most of the other edits are simple formatting.

### CODE REVIEW

File: specification/source/glossary.rst [hunk 1]
```diff
@@ -70,3 +85,13 @@ the Open Building Controls project.
    Mode
      In CDL, by mode we mean a signal that can take on multiple distinct
      values, such as ``On``, ``Off``, ``PreCool``.
+
+   Non-HVAC System
+     Any non-HVAC plant coupled with the control system.
+
+   Plant
```
Comment (JayHuLBL) [inline specification/source/glossary.rst]: @milicag Is this "Plant" interpretation aligning with the one shown in the ISO standard, or any other source?

Comment (milicag) [reply specification/source/glossary.rst]: No, the definition relies on our OBC requirements document. In ISO 16484 the plant definition is broader and includes "a dedicated functional units for controls". However, the way the new terms are aligned should be internally consistent in the scope of OBC project.

Comment (mwetter) [reply specification/source/glossary.rst]: We should also use "plant" for "a dedicated functional units for controls" as ISO does. This is customary in the control literature where you always have the separation between a "controller" and a "plant".

Maybe we should not use "non-HVAC system" but rather be specific, as a non-HVAC system could be an electrical system, an active facade, etc.

Comment (milicag) [reply specification/source/glossary.rst]: @mwetter I see some ambiguity in your comment. Let me see if I understand this correctly. Would you like us to use term ```plant``` and ```controller``` separately (which is how our documentation currently handles it, including this PR's addition to it), or would you prefer the term ```plant``` to include both the ```controller``` and the controlled plant?

There seems to be conflict within  ISO 16484-2, please see the ```plant``` definition and compare to a note related to the definition of the term ```system``` (all pasted below). In general, in the standard they start relatively clearly with the definition of physical and functional units, and build up from there.

One option that might fit both your and ISO definitions is that the term ```plant``` can be used for to describe either a physical unit - plant or the functional unit - controls.
 
```
3.149
plant
physical unit for a comprehensive process including the dedicated functional unit 
for control (3.51)

3.51
control
purposeful action on or in a process to meet specified objectives

3.187
system in its context a given arrangement of functional units such as 
equipment/devices, elements, and programs related to each other. 
Physical units can put functional units into effect cf. plant (3.149)
NOTE The definition indicates system as a functional unit, 
and plant as a physical unit.
```

Comment (mwetter) [reply specification/source/glossary.rst]: @milicag : I would use `controller` and `plant` as is used in https://en.wikibooks.org/wiki/Control_Systems/Feedback_Loops

**Comment by milicag** [comment]
@mwetter I've introduced those terms as advised.

**Comment by milicag** [comment]
@mwetter I see some comments on the usage of terms such as ```plant``` in Brent's and Matt's review. Maybe merging the glossary from this branch would help resolve ambiguities. I've merged the master in. The difference in use cases are some comments Phil has added.

File: specification/source/glossary.rst [hunk 2]
```diff
@@ -42,13 +48,22 @@ the Open Building Controls project.
...
+     or is untested. For example, a requirement would be that the actual
+     actuation signal is within 2% of the signal computed using the CDL
+     representation of a sequence, provided that they both receive the same
+     input data.
+
+   Control System
+     Any software and hardware required to perform the control function for a plant.
+     
+   Controller
+      A controller, or a compensator is an additional system that is added to the
```
Comment (mwetter) [inline specification/source/glossary.rst:61]: We don't use compensator, hence there is no need to add it.

File: specification/source/glossary.rst [hunk 3]
```diff
@@ -42,13 +48,22 @@ the Open Building Controls project.
...
+     input data.
+
+   Control System
+     Any software and hardware required to perform the control function for a plant.
+     
+   Controller
+      A controller, or a compensator is an additional system that is added to the 
+      plant to control the operation of the  plant. The system can have multiple 
+      compensators, and they can appear anywhere in the system: Before the pick-off node, 
+      after the summer, before or after the plant, and in the feedback loop.
```
Comment (mwetter) [inline specification/source/glossary.rst:64]: Adding a controller after the summer?

**Comment by mwetter** [comment]
I will merge it and then correct it. For example, adding a compensator after summer makes no sense?

**Pull request merged**